### DISCUSSION TITLE
failed to boot after flashing jetson orin nx

### DISCUSSION THREAD

**Discussion by maciej-narojczyk** [OE4T/meta-tegra#1307] [Q&A]
Hi,

the board is advantech EPC-R7300 with nvme 
I do not have a serial logs since i do not have the right cable yet but,
I flashed succesfully with nvidia L4T 35.3.1 scripts directly using:

sudo ./tools/kernel_flash/l4t_initrd_flash.sh --external-device nvme0n1p1 -c tools/kernel_flash/flash_l4t_external.xml -p “-c bootloader/t186ref/cfg/flash_t234_qspi.xml --no-systemimg” --showlogs --network usb0 p3768-0000+p3767-0000-nvme external

or

sudo ./tools/kernel_flash/l4t_initrd_flash.sh --external-device nvme0n1p1 -c tools/kernel_flash/flash_l4t_external.xml -p “-c bootloader/t186ref/cfg/flash_t234_qspi.xml --no-systemimg” --showlogs --network usb0 jetson-orin-nano-devkit-nvme external

with some hacks in l4t_flash_from_kernel.sh.
and it boots fine.

and I flashed succesfully  with sudo ./initrd using demo-image-base and the machine p3768-0000-p3767-0000
but it doesnt boot beyond UEFI.

since i dont have serial logs are there any obvious things to check to compare these images ?

thanks

**Comment by fabiocabaceira** [comment]
Hey,

Try building demo-image-full, flashing it, and connecting it to an HDMI screen to see if it works. The image you're currently building does not have a graphical interface, and that may be causing interference.
